Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -436,17 +486,16 @@ public string EncodeImage(string inputPath, DateTime dateModified, string output
             var width = newImageSize.Width;
             var height = newImageSize.Height;
 
-            using var resizedBitmap = new SKBitmap(width, height, bitmap.ColorType, bitmap.AlphaType);
-            // scale image
-            bitmap.ScalePixels(resizedBitmap, SKFilterQuality.High);
+            // scale image (the FromImage creates a copy)
+            using var resizedBitmap = SKBitmap.FromImage(ResizeImage(bitmap, new SKImageInfo(width, height, bitmap.ColorType, bitmap.AlphaType, bitmap.ColorSpace)));
```

Comment:
This is a tad more readable imo.
```suggestion
            var imageInfo = new SKImageInfo(width, height, bitmap.ColorType, bitmap.AlphaType, bitmap.ColorSpace);
            using var resizedBitmap = SKBitmap.FromImage(ResizeImage(bitmap, imageInfo));
```